index.js

```diff
@@ -11,7 +11,7 @@ const fn = (name, org) => {
     console.log('Organisation name is:', 'org_name');
   }
 
-  while (1 === 1) {
+  while (1 === 2) {
     console.log('Loop');
   }
```

Comment:
:robot: OpenAI

The while loop condition has been changed from `1 === 1` to `1 === 2`. This will cause the loop to never execute, effectively making it useless code. Please update the condition to a meaningful one or remove the loop altogether if it's not needed.
```suggestion
while (condition) {
   console.log('Loop');
}
```

<!-- This is an auto-generated comment by OpenAI -->